src/workloads/execution/HashedIndex.yml

```diff
@@ -0,0 +1,37 @@
+# This workload inserts ~1GB of documents in the first phase, and creates a hashed index on them in the second.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 500000
```

Comment:
I don't think you have 1 GB of data here. I'm guessing it's about 500MB since the string length is 1000.

Reply:
I changed the comment so that it's more generic